pkg/haprovider/haprovider.go

```diff
@@ -258,3 +296,7 @@ func (r *HAProvider) ensureEndpoints(ctx context.Context, serviceName, serviceNa
 	}
 	return endpoints, nil
 }
+
+func GetAviInfraSettingName(adc *akoov1alpha1.AKODeploymentConfig) string {
```

Comment:
The name is not very understandable here, better to change to something like: `<adc-name>-ais`

Reply:
Make sense to me!